MPF.md

```diff
@@ -1079,7 +1079,27 @@ Primarily for developer conveniance, this setting allows you to easily disable t
 }
 ```
 
-### Append to the main bundlesequence
+### Bundlesequence
+
+#### Classification bundles before autorun
+
+You can specify a list of bundles which should be run before autorun policies (if enabled).
+
+```json
+{
+  "vars":{
+    "control_common_bundlesequence_classification": [ "classification_one", "classification_two" ]
```

Comment:
I think the name is fine, could also use a name which describes exactly when the bundles are run, rather than what to put in them.

```
control_common_bundlesequence_pre_autorun
```

I think I prefer your name.